conf/messages

```diff
@@ -1141,6 +1141,21 @@ itemCommodityCode.defaultItem = Choose commodity code
 
 itemQuantity.title = How much {0} are you moving?
 itemQuantity.heading = How much {0} are you moving?
+itemQuantity.label = How much {0} are you moving?
+itemQuantity.hint.liquid = Enter the total quantity of liquids in {0}.
+
+itemQuantity.title.T200 = How many thousands of cigarettes are you moving?
```

Comment:
Shouldn't this match the h1, not the label? (i.e. "Number of cigarettes you are moving")

Reply:
It could do although I thought the title should have the same wording as the label **How many thousands of cigarettes are you moving** otherwise the title is going to be **Number of cigarettes you are moving**

I felt the nature of this ticket was to be more explicit that for T200's we are actually asking how many cigarettes expressed as thousands and that the title should match this explicitness.

Reply:
but it doesn't match the prototype so i'll change it

Reply:
Asked on the UCD channel

Reply:
UCD have said its needs to match the H1 as you have said so https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/394/commits/6bd06284620c60275b052a360b8886cadfa61491 puts that right